package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gamma-app/y-prosemirror",
-  "version": "1.2.0",
+  "version": "1.2.0-1",
```

Comment:
what is this versioning?  is this how we denote our own additions?

Reply:
Yep, that's how we've been denoting our changes on top of the b ase